### PR TITLE
Event schedule button bug fix

### PR DESCRIPTION
**Pull request by noahSerafin**
eventScheduleButton now uses eventId

### CODE REVIEW

File: src/components/ViewScheduleButton/__snapshots__/ViewScheduleButton.test.js.snap [hunk 1]
```diff
@@ -3,7 +3,7 @@
 exports[`should render the button 1`] = `
 <a
   class="view-schedule-button"
-  href="/event-schedule"
+  href="/undefined/event-schedule"
```
Comment (DanForder) [inline src/components/ViewScheduleButton/__snapshots__/ViewScheduleButton.test.js.snap:6]: make sure you pass the event id in this test so this isn't undefined

File: src/components/EventHeaderTitle/EventHeaderTitle.scss [hunk 2]
```diff
@@ -12,7 +12,6 @@
   padding-right: 1.438rem;
 
   &__subtitle {
-    font-size: 1rem;
```
Comment (DanForder) [inline src/components/EventHeaderTitle/EventHeaderTitle.scss:15]: Was removing this intentional?

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/EventHeaderTitle/EventHeaderTitle.scss (hunk 2 above)
Comment (noahSerafin) [reply src/components/EventHeaderTitle/EventHeaderTitle.scss:15]: I think so, it doesn't actually effect the size of the subtitle

File: src/components/ViewScheduleButton/ViewScheduleButton.test.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import ViewScheduleButton from "./ViewScheduleButton";
 import { customRender } from "../../utils/testUtils";
 
 it("should render the button", () => {
-  customRender(<ViewScheduleButton />);
+  customRender(<ViewScheduleButton eventId='eventId'/>);
```
Comment (DanForder) [inline src/components/ViewScheduleButton/ViewScheduleButton.test.js:6]: e.g. eventId={1234}

**Review by DanForder — CHANGES_REQUESTED** [review]

File: src/components/ViewScheduleButton/ViewScheduleButton.test.js (hunk 3 above)
Comment (noahSerafin) [reply src/components/ViewScheduleButton/ViewScheduleButton.test.js]: all done

**Review by DanForder — APPROVED** [review]

**Pull request merged**